redfish-core/lib/bios.hpp

```diff
@@ -161,11 +161,6 @@ inline void requestRoutesBiosService(App& app)
                 // Get the ActiveSoftwareImage and SoftwareImages
                 fw_util::populateFirmwareInformation(
                     asyncResp, fw_util::biosPurpose, "", true);
-                asyncResp->res.jsonValue["@Redfish.Settings"] = {
-                    {"@odata.type", "#Settings.v1_3_0.Settings"},
-                    {"SettingsObject",
-                     {{"@odata.id",
-                       "/redfish/v1/Systems/system/Bios/Settings"}}}};
                 asyncResp->res.jsonValue["AttributeRegistry"] =
```

Comment:
need to remove BMCWEB_ROUTE registered for Get support  right?

  BMCWEB_ROUTE(app, "/redfish/v1/Systems/system/Bios/Settings")
        .privileges({{"ConfigureManager"}})
        .methods(boost::beast::http::verb::get)

Reply:
Yeah, we should remove the GET and PATCH as well.

Reply:
GET is needed if a client is interested to see the Pending attributes on Bios - So i am not that sure if this can be removed.
PATCH is needed for a client to set the Bios attributes on the system - This must not be removed